tests/specs/test-transaction-sync.php

```diff
@@ -17,12 +21,72 @@ function setUp() {
 		) );
 
 		update_option( 'woocommerce_currency', 'USD' );
+
+		global $wpdb;
+		$wpdb->query( "DELETE FROM {$wpdb->prefix}taxjar_record_queue;" );
 	}
 
 	function tearDown() {
 		parent::tearDown();
 
 		WC_Taxjar_Record_Queue::clear_queue();
+
+		$this->clean_up_transactions_in_taxjar();
+	}
+
+	/**
+	 * Deletes all orders and refunds created in TaxJar
+	 */
+	function clean_up_transactions_in_taxjar() {
+		foreach( $this->synced_order_ids as $order_id ) {
+			$record = new TaxJar_Order_Record( $order_id );
+			$record->delete_in_taxjar();
+		}
+		$this->synced_order_ids = [];
+
+		foreach( $this->synced_refund_ids as $refund_id ) {
+			$record = new TaxJar_Refund_Record( $refund_id );
+			$record->delete_in_taxjar();
+		}
+		$this->synced_refund_ids = [];
+	}
+
+	/**
+	 * Creates a test order and adds order ID to array to be cleaned up during teardown
+	 * @param int $customer_id - customer ID for order
+	 * @param array $order_options - option array to override default test order options
+	 *
+	 * @return WC_Order|WP_Error
+	 */
+	function create_test_order( $customer_id = 1, $order_options = array() ) {
+		$order = TaxJar_Order_Helper::create_order( $customer_id = 1, $order_options = array() );
+		array_push($this->synced_order_ids, $order->get_id());
+		return $order;
+	}
+
+	/**
+	 * Creates a test refund and adds the refund ID to array to be cleaned up during teardown
+	 * @param $order_id - ID or order to create refund for
+	 *
+	 * @return WC_Order_Refund|WP_Error
+	 */
+	function create_test_refund( $order_id ) {
+		$refund = TaxJar_Order_Helper::create_refund_from_order( $order_id );
+		array_push($this->synced_refund_ids, $refund->get_id());
+		return $refund;
+	}
+
+	/**
+	 * Creates a test order with no tax and adds the order ID to array to be cleaned up during teardown
+	 * @param int $customer_id - customer ID for order
+	 * @param array $order_options - option array to override default test order options
+	 *
+	 * @return WC_Order|WP_Error
+	 */
+	function create_test_order_with_no_tax( $customer_id = 1, $order_options = array() ) {
+		$order = TaxJar_Order_Helper::create_order_with_no_tax( $customer_id = 1, $order_options = array() );
+		array_push($this->synced_order_ids, $order->get_id());
```

Comment:
Not sure I fully understand the spacing convention, but should there be spaces around the arguments in the calls to `array_push`? If so, there are a couple other instances in these changes you can find by searching the project.